test/test_component_selector.jl

```diff
@@ -0,0 +1,206 @@
+# TODO copied directly from https://github.com/GabrielKS/PowerAnalytics.jl/tree/gks/entity-metric-redesign, will require major refactor
+
+test_sys = PSB.build_system(PSB.PSITestSystems, "c_sys5_all_components")
+test_sys2 = PSB.build_system(PSB.PSISystems, "5_bus_hydro_uc_sys")
+
+sort_name(x) = sort(collect(x); by = get_name)
+
+@testset "Test helper functions" begin
+    @test subtype_to_string(ThermalStandard) == "ThermalStandard"
+    @test component_to_qualified_string(ThermalStandard, "Solitude") ==
+          "ThermalStandard__Solitude"
+end
```

Comment:
Why was the third test
```Julia
@test component_to_qualified_string(
        PSY.get_component(ThermalStandard, test_sys, "Solitude"),
    ) == "ThermalStandard__Solitude"
```
removed from here?

Reply:
This should be added back,